cmd/minishift/cmd/openshift/config.go

```diff
@@ -27,8 +27,8 @@ const (
 
 var configCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
-	Short: "Displays or patches OpenShift configuration.",
-	Long:  "Displays or patches OpenShift master or node configuration. Patches are supplied in JSON format.",
+	Short: "Displays or updates the OpenShift configuration.",
```

Comment:
I don't think so. At one point we are saying "updating .... with a patch", but this still keeps the reference to patching. Patching means more than updating. Updating can happen in many ways, patching is the process of updating something (config, software, etc) using the diff format.

I am pretty sure we had this discussion before when eating the asciidoctor docs.